ports/mimxrt10xx/reset.c

```diff
@@ -35,5 +35,5 @@ void reset(void) {
 }
 
 bool bootloader_available(void) {
-    return &_bootloader_dbl_tap >= 0;
+    return SNVS->LPGPR[0] >= 0;
```

Comment:
Don't think this logic works, the previous code was checking the address of `_bootloader_dbl_tap`, if it was 0 then there was no bootloader and if it was at the end of stack then there was a bootloader, the new version just checks a register value.

Reply:
I think checking the register value will actually work once the bootloader sets it for its own use. Although it is preserved on reset, I believe it is zeroed on power up.